community/neo4j/src/docs/ops/upgrades.asciidoc

```diff
@@ -4,89 +4,77 @@
-These upgrades are either automatic, or require explicit configuration to allow them.
-Upgrades between patch releases within the same minor version are automatic.
-Upgrading to the latest patch release is recommended before upgrading to new major or minor versions.
-
-A database created by an older version of Neo4j will be upgraded during startup when opened by the target version of Neo4j.
-
-The following Neo4j versions, where _x_ stands for the latest patch release, can be upgraded to Neo4j {neo4j-version}:
-
-1.9.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
-
-2.0.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
-
-2.1.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
-
-2.2.x -> {neo4j-version}::
-<<explicit-upgrade,Explicit configuration is required>>
+[CAUTION]
+====
+Before starting any upgrade actions, it is very important to make a backup of the Neo4j database.
+If using the 
+ifndef::upgradetext[<<operations-backup, online backup tool>>, ]
+ifdef::upgradetext[online backup tool (see http://neo4j.com/docs/{neo4j-version}/operations-backup.html), ]
+available with Neo4j Enterprise, ensure that backups have completed successfully.
+====
 
 [NOTE]
-Downgrade is only supported between Neo4j versions that allow for automatic store upgrades.
-This typically means only within patch releases of the same Neo4j version.
-
-[[automatic-upgrade]]
-== Automatic Store Upgrade
-
-To perform an automatic store upgrade:
-
-. Cleanly shut down the older version of Neo4j, if it is running.
+====
+Throughout this instruction, the files used to store the Neo4j data are referred to as `database files`.
+The location of these files is specified by configuring the `org.neo4j.server.database.location` variable in `conf/neo4j-server.properties`.
+====
 
-. Install Neo4j {neo4j-version}, and set it up to use the same database store directory (typically _data/graph.db_).
+[[supported-upgrade-paths]]
+== Supported upgrade paths
 
-. Make a copy of the database.
-+
-[IMPORTANT]
-It is strongly advised to make a copy of the database store directory at this time, to use as a backup in case rollback/downgrade is required.
-This is not necessary if a backup has been made using the
-ifndef::upgradetext[<<operations-backup, online backup tool>>, ]
-ifdef::upgradetext[online backup tool (see http://neo4j.com/docs/{neo4j-version}/operations-backup.html), ]
-available with Neo4j Enterprise.
+Before upgrading to a new major or minor release, the Neo4j database must first be upgraded to the latest patch release within the relevant major and minor release.
+You can find out the relevant patch release number for your installation at this page: {neo4j-releases-download-page}.
+The following Neo4j upgrade paths are supported, where _x_ stands for the latest patch release within a specific minor release, and _y_ stands for any patch release number:
```

Comment:
Could we say (e.g.) "1.9.latest" and "2.3.any" (or something like that) rather than defining variables. People glancing at the page will only see the list and it's easy to overlook the difference between "x" and "y".